integration-tests/src/test/burn.test.ts

```diff
@@ -0,0 +1,121 @@
+import { Blockchain, KeyringPair, creditcoinApi, BN, CREDO_PER_CTC } from 'creditcoin-js';
+import { forElapsedBlocks, testData } from 'creditcoin-js/lib/testUtils';
+import { CreditcoinApi } from 'creditcoin-js/lib/types';
+import { mnemonicGenerate } from '@polkadot/util-crypto';
+import { extractFee, describeIf } from '../utils';
+
+describeIf((global as any).CREDITCOIN_EXECUTE_SETUP_AUTHORITY, 'burn', () => {
+    let ccApi: CreditcoinApi;
+    let sudoSigner: KeyringPair;
+    let wallet: KeyringPair;
+
+    const testingData = testData(
+        (global as any).CREDITCOIN_ETHEREUM_CHAIN as Blockchain,
+        (global as any).CREDITCOIN_CREATE_WALLET,
+    );
+    const { keyring } = testingData;
+
+    const ONE_CTC = new BN((1 * CREDO_PER_CTC).toString(), 10);
+    const POINT_01_CTC = new BN((0.01 * CREDO_PER_CTC).toString(), 10);
+
+    beforeAll(async () => {
+        ccApi = await creditcoinApi((global as any).CREDITCOIN_API_URL);
+        sudoSigner = (global as any).CREDITCOIN_CREATE_SIGNER(keyring, 'sudo');
+        wallet = (global as any).CREDITCOIN_CREATE_SIGNER(keyring, 'borrower');
```

Comment:
`wallet` variable is only used in 1 scenario and shouldn't be global to all of them. It is in essence what you call `burner` in all of the other scenarios.

Please move it as a local variable for `it('burn_all works as expected'`.